pom.xml

```diff
@@ -93,7 +92,13 @@
         <groupId>org.mule.runtime</groupId>
         <artifactId>mule-dwb-api</artifactId> 
         <scope>provided</scope>         
-    </dependency> 
+    </dependency>
+    <dependency>
+      <groupId>org.mule.tests</groupId>
+      <artifactId>test-components</artifactId>
+      <version>${project.version}</version>

```

Comment:
if this is a new repo why not to use version 1.0.0? are you thinking about to branch it in the same way of the runtime projects with master, 4.3.x etc in the future?

Reply:
No, actually the connector has been created to test new features added in 4.4.0 for MTS.